tools/rosbag_storage/include/rosbag/bag.h

```diff
@@ -569,7 +581,7 @@ void Bag::doWrite(std::string const& topic, ros::Time const& time, T const& msg,
 
         // Check if we want to stop this chunk
         uint32_t chunk_size = getChunkOffset();
-        logDebug("  curr_chunk_size=%d (threshold=%d)", chunk_size, chunk_threshold_);
+        CONSOLE_BRIDGE_logDebug("  curr_chunk_size=%d (threshold=%d)", chunk_size, chunk_threshold_);
```

Comment:
Oh, sorry, I missed that.  I got confused because the initial bug report was for Indigo.

Alright, so for Lunar, I agree with your original patch (I'll revert mine off of this branch).  However, that still leaves the original bug report about Indigo.  It sounds like you want to just leave this be on Indigo; is that the case?  If so, once we merge this patch I'll close the original bug report saying that it won't be fixed back in Indigo, but will be fixed going forward.  Sound reasonable?